tests/test_interconnect/test_interconnect_cgra.py

```diff
@@ -32,6 +32,17 @@ def dw_files():
     return result_filenames
 
 
+def test_1x1():
+    # this is all PE
+    interconnect = create_cgra(1, 1, IOSide.None_, num_tracks=3, mem_ratio=(0, 1))
+    circuit = interconnect.circuit()
+    with tempfile.TemporaryDirectory() as temp:
+        filename = os.path.join(temp, "1x1")
+        import magma
+        magma.compile(filename, circuit, output="coreir-verilog")
+        assert os.path.isfile(filename + ".v")
+
+
```

Comment:
LGTM, doesn't really test functionality of the generated circuit but I think we can use https://github.com/StanfordAHA/garnet/pull/697 to do that so this is fine for now.